org.gnome.Podcasts.json

```diff
@@ -8,17 +8,11 @@
     ],
     "command" : "gnome-podcasts",
     "finish-args" : [
-        "--filesystem=xdg-run/dconf",
-        "--filesystem=~/.config/dconf:ro",
-        "--talk-name=ca.desrt.dconf",
-        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
         "--share=network",
         "--share=ipc",
-        "--socket=x11",
+        "--socket=fallback-x11",
         "--socket=wayland",
-        "--socket=pulseaudio",
-        "--device=dri",
```

Comment:
Why was this removed?

Reply:
Was planned for accessing gl with the `gstglsink` but plans changed and it won't be needed.